compile.sh

```diff
@@ -924,6 +940,10 @@ RANLIB=$RANLIB CFLAGS="$CFLAGS $FLAGS_LTO" CXXFLAGS="$CXXFLAGS $FLAGS_LTO" LDFLA
 --with-yaml \
 --with-openssl \
 --with-zip \
+--with-libbson="yes" \
+--with-libmongoc="yes" \
+--with-mongodb-system-libs="yes" \
+--with-mongodb-ssl="auto" \
```

Comment:
mongodb hasn't been downloaded at this point ...

Reply:
normally i put the driver on windows it should work and then on mac i found the line that asks but i don't know the rule.
this is line 1072 in compile.sh who create problem au mac compilation